### PR TITLE
Sensor streams changes

### PR DESCRIPTION
**Pull request by menaka121**

### CODE REVIEW

File: components/device-mgt-iot-androidsense/org.wso2.carbon.device.mgt.iot.androidsense.api/src/main/java/org/wso2/carbon/device/mgt/iot/androidsense/service/impl/AndroidSenseService.java [hunk 1]
```diff
@@ -260,89 +260,85 @@ public void pushSensorData(
...
-                default:
-                    try {
-                        int androidSensorId = Integer.parseInt(sensor.key);
-
-                        if (androidSensorId == 2) {
-                            streamDef = MAGNETIC_STREAM_DEFINITION;
-                            String value = sensor.value;
-                            String valuesM[] = value.split(",");
-                            Float gValuesF[] = new Float[1];
+            if(sensor.key.equals("battery")){
```
Comment (sinthuja) [inline components/device-mgt-iot-androidsense/org.wso2.carbon.device.mgt.iot.androidsense.api/src/main/java/org/wso2/carbon/device/mgt/iot/androidsense/service/impl/AndroidSenseService.java]: We need to add the battery, gps, etc as constants.

File: features/device-mgt-iot-androidsense-feature/org.wso2.carbon.device.mgt.iot.androidsense.analytics/Light/Eventstream_light_1.0.0/org.wso2.iot.devices.light_1.0.0.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "org.wso2.iot.devices.light",
   "version": "1.0.0",
-  "nickName": "Light Data",
-  "description": "Light data received from the Device",
+  "nickName": "light Data",
```
Comment (sinthuja) [inline features/device-mgt-iot-androidsense-feature/org.wso2.carbon.device.mgt.iot.androidsense.analytics/Light/Eventstream_light_1.0.0/org.wso2.iot.devices.light_1.0.0.json]: Let's have this change, and let's have the sentence starts with upper case.

File: components/device-mgt-iot-androidsense/org.wso2.carbon.device.mgt.iot.androidsense.api/src/main/java/org/wso2/carbon/device/mgt/iot/androidsense/service/impl/AndroidSenseService.java (hunk 1 above)
Comment (menakaj) [reply components/device-mgt-iot-androidsense/org.wso2.carbon.device.mgt.iot.androidsense.api/src/main/java/org/wso2/carbon/device/mgt/iot/androidsense/service/impl/AndroidSenseService.java]: This class is not used now as the structure of the device plugin is changed. Hence this class has been removed from the repository.

File: features/device-mgt-iot-androidsense-feature/org.wso2.carbon.device.mgt.iot.androidsense.analytics/Light/Eventstream_light_1.0.0/org.wso2.iot.devices.light_1.0.0.json (hunk 2 above)
Comment (menakaj) [reply features/device-mgt-iot-androidsense-feature/org.wso2.carbon.device.mgt.iot.androidsense.analytics/Light/Eventstream_light_1.0.0/org.wso2.iot.devices.light_1.0.0.json]: This file is removed from the repo as the device type changed.

**Pull request merged**